common/BUILD

```diff
@@ -0,0 +1,21 @@
+load("//tools:ts_library.bzl", "ts_library")
+load("//tools:expand_template.bzl", "expand_template")
+load("//:version.bzl", "DF_VERSION")
+
+package(default_visibility = ["//visibility:public"])
+
+expand_template(
+    name = "version",
+    out = "version.ts",
+    substitutions = {
+        "$DF_VERSION": DF_VERSION,
+    },
+    template = "version.ts.tmpl",
+)
+
+ts_library(
+    name = "current_version",
+    srcs = [
+        ":version.ts",
+    ]
+)
```

Comment:
nit: either output `current_version.ts` from the above rule, or simply rename this to `version` and change the rule above to `version_tmpl_expansion`